README.md

```diff
@@ -104,8 +93,8 @@ to `./runner.yml` and override relevant properties.
 
 To set up the project run:
 
-```
-$ ./vendor/bin/run drupal:site-setup
+```bash
```

Comment:
This should be removed like in other repositories we are now updating.

Reply:
Fixed.